### PR TITLE
feat: Create a component for a new board

### LINKED ISSUE #16: [Client] [Boards] Create board box component
Board box should be able to display:
1. Board name 
2. Created by

image

### PR DESCRIPTION
**Pull request by pratiyush2706**
Resolves #16

### CODE REVIEW

File: client/src/App.test.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+import ReactDOM from 'react-dom';
+import App from './App';
 import SignUp from './containers/SignUp/SignUp';
-import { shallow } from "enzyme";
+import { shallow } from 'enzyme';
 
-describe("App component", () => {
-  it("Test Render component using shallow", () => {
+describe('App component', () => {
+  it('Test Render component using shallow', () => {
```
Comment (rishabh1403) [inline client/src/App.test.js]: bad test message

File: client/src/App.test.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     const wrapper = shallow(<App />);
     expect(wrapper.find(SignUp)).toHaveLength(1);
   });
-});
 
-it("renders without crashing", () => {
-  const div = document.createElement("div");
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+  it('renders without crashing', () => {
```
Comment (rishabh1403) [inline client/src/App.test.js]: bad test message

File: client/src/components/Board/BoardBox.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.board-container {
+    background-color: lightgrey;
+    border: 1px solid black;
+    box-shadow: 2px;
+    border-radius: 5px;
+    width: 150px;
+    height: 70px;
+}
```
Comment (rishabh1403) [inline client/src/components/Board/BoardBox.css]: newline :)

**Comment by rishabh1403** [comment]
seems like a UI change, please add a screenshot
add better pr name

**Comment by pratiyush2706** [comment]
Screenshot:
Screenshot_20190607_004132

**Comment by rishabh1403** [comment]
> seems like a UI change, please add a screenshot
> add better pr name

@pratiyush2706 please request review after you have resolved all issues, i can see pr name still not following the standards

**Comment by pratiyush2706** [comment]
I did re-request a review after I made the changes.

**Comment by pratiyush2706** [comment]
I have also made changes to the PR name now.

**Pull request merged**